SeaSharpe_CVGS/SeaSharpe_CVGS/Controllers/GameController.cs

```diff
@@ -12,7 +15,15 @@ public class GameController : Controller
     {
         private ApplicationDbContext db = new ApplicationDbContext();
 
+        //Dictionary containing ESRB ratings
+        public static List<string> esrbList = new List<string>
+            {
+                {"EC"},{"E"},{"E10"},{"T"},{"M"},{"AO"}
```

Comment:
The comment says "Dictionary" but this is a list. A dictionary might be useful so that we can display more helpful text to the users like I did for Gender: 

```csharp
        public static Dictionary<string, string> genders = new Dictionary<string, string>() { { "Male", "M" }, {"Female", "F"}, {"Other", "O"} };
        // Using a dictionary we can create a select list like this:
        IEnumerable<System.Web.Mvc.SelectListItem> genderSelectList= 
            from gender in genders 
            select new System.Web.Mvc.SelectListItem 
            { Text = gender.Key, Value = gender.Value };
```